### ISSUE TITLE
UI: Add ability to change/add/delete the Alias of a Contact

### ISSUE THREAD

**Issue by paskalito** [ChatSecure/ChatSecure-iOS#786]
Currently the only way to add an alias to a contact is when you add the user yourself.

No way to change it later.
more importantly also no way to set one if you get added by the other side.

**Comment by chrisballinger** [comment]
Good call

**Comment by paskalito** [comment]
thank you :) nice to hear you agree.
in terms of motivation / why i think we need this it's besically the same as i wrote here #757

**Comment by Echolon** [comment]
Yes, just yesterday a friend of mine tried to rename me (as I added her!) :(
